pkg/nodeset/v1alpha1/types.go

```diff
@@ -59,6 +61,9 @@ type NodeClassResource struct {
 	Reference *v1.ObjectReference `json:"reference,omitempty"`
 }
 
+// +genclient=true
+// +nonNamespaced=true
```

Comment:
Wait, we set `nonNamespaced` to true. Does this work with client-gen and the old TPRs?